README.md

```diff
@@ -16,9 +16,9 @@ See the [Equinix Metal Actions Example](https://github.com/equinix-labs/metal-ac
 
 With | Environment variable | Description
 --- | --- | ---
-`userToken` | `METAL_AUTH_TOKEN` | (required) A Equinix Metal User API Token
+`authToken` | `METAL_AUTH_TOKEN` | (required) A Equinix Metal User API Token
 `projectID` | `METAL_PROJECT_ID` | (required) Project ID that will be deleted.
-`keepProject` | `KEEP_PROJECT` | When set to the default of `false`, the project will be deleted after all project contents are deleted.
+`keep` | `KEEP_PROJECT` | When set to the default of `false`, the project will be deleted after all project contents are deleted.
```

Comment:
It's possible `keepProject` was correct in practice.  Or maybe this hasn't been working for a while?
In any case, the input is referred to as `keep` in the [`inputs` section](https://github.com/ctreatma/metal-sweeper-action/blob/main/action.yml#L6), but the [`runs` section is looking at `keepProject` to populate the environment variable](https://github.com/ctreatma/metal-sweeper-action/blob/main/action.yml#L21)

Reply:
I think you're right. It needs to match in both locations to be usable. I'll amend the PR to align the name down the stack.